CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+- Fix `abi3-py310` feature implementation: use Python 3.10 ABI when available. [#2242](https://github.com/PyO3/pyo3/pull/2242)
```

Comment:
Maybe specific the symptoms of the error, e.g. "instead of silently falling back to the 3.9 ABI"?

Reply:
Updated.

Reply:
Thanks!